doc/api/test.md

```diff
@@ -449,6 +460,9 @@ added: v18.0.0
   * `todo` {boolean|string} If truthy, the test marked as `TODO`. If a string
     is provided, that string is displayed in the test results as the reason why
     the test is `TODO`. **Default:** `false`.
+  * `timeout` {number} A number of milliseconds the test will fail after.
+    If unspecified, subtests inherit this value from their parent.
+    **Default:** `30_000`.
```

Comment:
The second line says subtests inherit this value when unspecified, but then the documentation also gives a default value. This seems contradictory.

Reply:
the default is 30000, but if you override that - it will override for all nested tests

Reply:
We use `Default:` to refer to the default value if not set. But here, it seems that the default is whatever is inherited.